cli/azd/pkg/tools/github/github.go

```diff
@@ -323,3 +377,185 @@ func isEnvVarTokenSource(message string) bool {
 
 	return false
 }
+
+func extractFromZip(src, dst string) (string, error) {
```

Comment:
Given that we've increased the LoC of `github,go` by 2x just for download, consider moving it to `download.go` or `acquire.go`.

Reply:
I don't think it is worth it, as this code will be eventually gone anyway.